web/src/utils/viewport.ts

```diff
@@ -4,11 +4,12 @@ import { useCallback, useState } from 'react';
 export function useReferenceWidthHeightObserver(offsetX = 0, offsetY = 0) {
   const [width, setWidth] = useState(0);
   const [height, setHeight] = useState(0);
-  const [node, setNode] = useState(null); // The DOM node
+  const [node, setNode] = useState<any>(null); // The DOM node
```

Comment:
could be <Element | null>?

Reply:
Good point - I had to update the weather layers and add some checks for this to work, but seems like they still work 🙈